apps/web/src/pages/activities/services/chart-bar/option.service.ts

```diff
@@ -83,6 +83,7 @@ function getTooltipConfiguration() {
         tooltipEl.id = 'chartjs-tooltip';
         tooltipEl.innerHTML = '<table></table>';
         document.body.appendChild(tooltipEl);
+        tooltipEl.style.position = 'absolute';
```

Comment:
this was a super weird issue when you navigate to the activity feed, we do show there a tooltip on the bar chart and it's created and inserted into the dom, and if the user navigates to the workflow page and the tour hints were shown it caused the layout changes... in the screenshot you can see that the hint caused layout shift but also when I fixed that the tooltip also cause that shift as it takes the space

![Screenshot 2023-08-11 at 01 07 43](https://github.com/novuhq/novu/assets/2607232/2106169b-73e6-4a3b-892c-bd606da49f6c)